pom.xml

```diff
@@ -706,5 +706,16 @@
 				</plugins>
 			</reporting>
 		</profile>
+		<profile>
+			<activation>
+				<file>
+					<exists>copyrightTemplateFile.txt</exists>
```

Comment:
Toch een beetje raar om aan de hand van deze file te bepalen of het de root folder is.